api/webhook/taskcluster.go

```diff
@@ -264,9 +274,12 @@ func createAllRuns(log shared.Logger,
 		go func(browser string, urls []string) {
 			defer wg.Done()
 			log.Infof("Reports for %s: %v", browser, urls)
-			err := createRun(client, sha, api, username, password, urls, labels)
+			err := createRun(client, sha, uploadURL, username, password, urls, labels)
 			if err != nil {
 				errors <- err
+			} else {
+				browserName := strings.Split(browser, "-")[0] // chrome-dev => chrome
+				suitesAPI.PendingCheckRun(sha, browserName)
```

Comment:
Do you have a rough idea how long this API call may take?

Reply:
Anecdotally, requests are quick (mostly <100ms, no total request times for requests that send these are over 1s)